.gitignore

```diff
@@ -3,6 +3,9 @@
 
 # Python
 __pycache__/
+*.pyc
+src/
```

Comment:
I think so. I'm, not that familiar with Python, but it contains a text file called `pip-delete-this-directory.txt`. So I guessed that there was no reason to commit it to the repo.